python/notes.py

```diff
@@ -196,14 +200,15 @@ def get_png(self, highlighted_frames):
                 else:
                     return None
             else:
-                return Image.open(self.circle_highlighted_pngs[min(highlighted_frames[0]-1,len(self.circle_highlighted_pngs))])
+                return Image.open(self.circle_highlighted_pngs[min(highlighted_frames[0],len(self.circle_highlighted_pngs))-1])
```

Comment:
there is merge conflict with these three, for the hotfix

Reply:
the current code in beta is:

```
return Image.open(self.root_highlighted_pngs[min(highlighted_frames[0],len(self.root_highlighted_pngs))-1])
```

and the current code in master is:
```
return Image.open(self.root_highlighted_pngs[min(highlighted_frames[0]-1,len(self.root_highlighted_pngs)-1)])
>>>>>>> master
```

is the second one the intended one after the hot fix? https://github.com/sky-music/sky-python-music-sheet-maker/commit/febb7d760c3be2240f0df4d77d20df7a6b2865b1